Classes/Aggregate/NewContentElementWizardAggregate.php

```diff
@@ -127,5 +169,23 @@ protected function processElement(array $element)
 
 EOS
         );
+
+        if(!in_array($iconName, $this->iconNames)) {
+            $extensionConfiguration = unserialize($GLOBALS['TYPO3_CONF_VARS']['EXT']['extConf']['mask_export']);
+            if (!empty($extensionConfiguration['exportIcons'])) {
+                $contentElementIcon =
+<<<ICON
+\$iconRegistry->registerIcon(
+    '$iconName',
+    \TYPO3\CMS\Core\Imaging\IconProvider\FontawesomeIconProvider::class,
+    [
+        'name'     => '$iconName'
+    ]
+);
+ICON;
```

Comment:
This code should be added on modifying the ext_localconf.php - Please only store iconName in the property. Not need to check for an existing one as same name should mean same icon